packages/nestjs-trpc/lib/decorators/__tests__/middlewares.decorator.spec.ts

```diff
@@ -4,24 +4,28 @@ import { MIDDLEWARES_KEY } from '../../trpc.constants';
 import { MiddlewareOptions, MiddlewareResponse, TRPCMiddleware } from '../../interfaces';
 
 describe('UseMiddlewares Decorator', () => {
-  const mockMiddleware = () => ({});
+  class TestMiddleware implements TRPCMiddleware {
+    use(opts: MiddlewareOptions<object>): MiddlewareResponse | Promise<MiddlewareResponse> {
+      throw new Error('Method not implemented.');
+    }
+  }
```

Comment:
[note] The typing enforces a class now so I moved all the tests to use this convention

Feel free to resolve